sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2KeyspacesResourceImpl.java

```diff
@@ -73,15 +70,14 @@ public Response getAllKeyspaces(
             .from("system_schema", "keyspaces")
             .build();
 
-    QueryOuterClass.Query query =
-        QueryOuterClass.Query.newBuilder().setParameters(paramsB.build()).setCql(cql).build();
+    QueryOuterClass.Query query = QueryOuterClass.Query.newBuilder().setCql(cql).build();
     QueryOuterClass.Response grpcResponse = blockingStub.executeQuery(query);
 
     final QueryOuterClass.ResultSet rs = grpcResponse.getResultSet();
 
     // two-part conversion: first from proto to JsonNode for easier traversability,
```

Comment:
does this comment still apply?

Reply:
Technically yes, but I Could probably change it or just remove.